### ISSUE TITLE
[Accessibility]: Multiple elements with "navigation" role do not have unique labels

### ISSUE THREAD

**Issue by yashgarg1719** [IBM/carbon-components-angular#2187]
## Detailed description

> Describe in detail the issue you're having.

When there are multiple elements with role "nav" on a page, they must have unique labels that clearly describe their purpose. This helps people who use assistive technologies to quickly understand and navigate to each navigation region.

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug

> Is this issue related to a specific component?

`<nav role="navigation">`

> What did you expect to happen? What happened instead? What would you like to see changed?

For each element with a "nav" role, either set the value of the aria-labelledby attribute to the id value of any visible text that appropriately and uniquely describes that navigation region; Or, use the aria-label attribute to add an invisible, unique, non-empty label to an element with the "nav" role.

> What browser are you working in?

Firefox 101.0(64-bit)

> What version of the Carbon Design System are you using?

`carbon-components@10.5.0`
`carbon-components-angular@4.56.4`

> What WCAG 2.1 checkpoint does the issue violate?

IBM 2.4.1 Bypass Blocks WCAG technique ARIA6 WCAG technique ARIA13

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

## Additional information

* Screenshots or code
![image](https://user-images.githubusercontent.com/108860673/178150713-efea8c82-5f0e-43e5-b4df-6d5232f9f9a4.png)

**Comment by Akshat55** [comment]
Hi @yashgarg1719, the user is responsible for setting `aria-label` (**ariaLabel**) or `aria-labelledby` (**ariaLabelledby**) in Tabs.

`<ibm-tabs>` (**Tabs**) accepts both values as input properties. You can observe this in the [documentations](https://angular.carbondesignsystem.com/?path=/story/components-tabs--documentation).

For `<ibm-breadcrumb>` (**Breadcrumbs**), an `ariaLabel` property is available as can be observed in the [documentation](https://angular.carbondesignsystem.com/?path=/story/components-breadcrumb--documentation) under `Inputs`.

Closing issue as issue can be resolved via input property. Feel free to reopen the issue or comment with more details to reopen.
